<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsManager</class>
 <widget class="QMainWindow" name="ContactsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Contacts</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="contacts_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>651</width>
      <height>381</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <property name="horizontalHeaderLabels" stdset="0">
     <stringlist/>
    </property>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Téléphone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_quitter">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>570</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="edit_button">
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="delete_button">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="name_input">
       <property name="placeholderText">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="prenom_input">
       <property name="placeholderText">
        <string>Prénom (facultatif)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="telephone_input">
       <property name="placeholderText">
        <string>Téléphone (facultatif)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="email_input">
       <property name="placeholderText">
        <string>Email (facultatif)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
